### PR TITLE
Add different color palette for Explore Timeline

### PR DESCRIPTION
**Pull request by Rperry2174**
Not really sure why the colors are different depending on the tag:
Screen Shot 2022-08-16 at 5 15 06 PM
Screen Shot 2022-08-16 at 5 14 55 PM

it should be in this order:
image

Started with this array and then tried to narrow it down to colors that wont clash, but also wont look too similar (and are equally spread out -- i.e. no two greens next to each other)

```
$clr1: rgb(115, 191, 105);
$clr3: rgb(242, 204, 12);
$clr4: rgb(138, 184, 255);
$clr5: rgb(255, 120, 10);
$clr6: rgb(242, 73, 92);
$clr7: rgb(87, 148, 242);
$clr8: rgb(184, 119, 217);
$clr9: rgb(112, 93, 160);
$clr10: rgb(55, 135, 45);
$clr11: rgb(250, 222, 42);
$clr12: rgb(68, 126, 188);
$clr13: rgb(193, 92, 23);
$clr14: rgb(137, 15, 2);
$clr15: rgb(10, 67, 124);
$clr16: rgb(109, 31, 98);
$clr17: rgb(88, 68, 119);
$clr18: rgb(183, 219, 171);
$clr19: rgb(244, 213, 152);
$clr20: rgb(112, 219, 237);
$clr21: rgb(249, 186, 143);
$clr22: rgb(242, 145, 145);
$clr23: rgb(130, 181, 216);
$clr24: rgb(229, 168, 226);
$clr25: rgb(174, 162, 224);
$clr26: rgb(98, 158, 81);
$clr27: rgb(229, 172, 14);
$clr28: rgb(100, 176, 200);
$clr29: rgb(224, 117, 45);
$clr30: rgb(191, 27, 0);
$clr31: rgb(10, 80, 161);
$clr32: rgb(150, 45, 130);
$clr33: rgb(97, 77, 147);
$clr34: rgb(154, 196, 138);
$clr35: rgb(242, 201, 109);
$clr36: rgb(101, 197, 219);
$clr37: rgb(249, 147, 78);
$clr38: rgb(234, 100, 96);
$clr39: rgb(81, 149, 206);
$clr40: rgb(214, 131, 206);
$clr41: rgb(128, 110, 183);
$clr42: rgb(63, 104, 51);
$clr43: rgb(150, 115, 2);
$clr44: rgb(47, 87, 94);
$clr45: rgb(153, 68, 10);
$clr46: rgb(88, 20, 12);
$clr47: rgb(5, 43, 81);
$clr48: rgb(81, 23, 73);
$clr49: rgb(63, 43, 91);
$clr50: rgb(224, 249, 215);
$clr51: rgb(252, 234, 202);
$clr52: rgb(207, 250, 255);
$clr53: rgb(249, 226, 210);
$clr54: rgb(252, 226, 222);
$clr55: rgb(186, 223, 244);
$clr56: rgb(249, 217, 249);
$clr57: rgb(222, 218, 247);
```

### CODE REVIEW

File: packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/colorPalette.ts [hunk 1]
```diff
@@ -56,3 +56,38 @@ export const ColorBlindPalette: FlamegraphPalette = {
   neutralColor: Color.rgb(148, 142, 142),
   badColor: Color.rgb(220, 50, 32),
 };
+
+export const TimelineSeriesPalette: FlamegraphPalette = {
```
Comment (eh-am) [inline packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/colorPalette.ts:60]: I think this is just a poc (based on the pr title), and you won't merge this yet

but I think it's important to say that I don't think this should belong here. The flamegraph component itself has nothing to do with the timeline.  So I recommend moving this to as closest as possible to where it's used (TagExplorer component most likely).

Comment (Rperry2174) [reply packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/colorPalette.ts:60]: yeah I was going to put it here but I do like these colors better for the timeline so I'll move it closer to there

**Review by eh-am — APPROVED** [review]

**Pull request merged**